iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/DanglingLineImpl.java

```diff
@@ -309,6 +308,7 @@ public void remove() {
             throw new UnsupportedOperationException("Parent tie line " + tieLine.getId() + " should be removed before the child dangling line");
         }
         super.remove();
+        boundary.getReferrerManager().notifyOfRemoval();
```

Comment:
We could delegate this to a `boundary.remove()` method

Reply:
Done